src/lib/libhtml5.js

```diff
@@ -2374,6 +2374,12 @@ var LibraryHTML5 = {
     return requestAnimationFrame(tick);
   },
 
+  emscripten_queue_microtask: (cb, userData) => {
+    queueMicrotask(() => {
+      {{{ makeDynCall('vp', 'cb') }}}(userData);
```

Comment:
This probably needs to be wrapped in `callUserCallback`.

Reply:
We do need `callUserCallback` whenever we enter user code from the event loop otherwise lots of things can break.   A classic one is calling `exit()` from within the callback.

IIRC the overhead of `callUserCallback` in MINIMAL_RUNTIME, where we don't have the same requirements is very close to zero.   If you want to make reduce it to absolutely zero that seems like a good thing, but unrelated to this specific change.